site/data/authors.yml

```diff
@@ -36,32 +36,32 @@ david:
 jordan:
   name: Jordan GS
   picture: /img/authors/zap_400x400.png
-  twitter: 
+  twitter:
```

Comment:
Would be good to revert these changes.

Reply:
My IDE (Vim) is configured to remove all trailing space characters upon saving. I thought it wouldn't be a problem to have them removed but I can revert it if you prefer.

Reply:
The problem is not removing trailing spaces, the problem is doing that in the same commit/PR as other changes.

Reply:
Makes sense, I understand. I like to have one commit per change too. I just thought it didn't matter for trailing spaces.

Done in 3a8e842.

Reply:
Could you create a PR?

Reply:
You mean for removing the trailing spaces? Sure, of course.

Sorry I misunderstood that you were interested in doing so but in another PR. I thought you just wanted to let it as it is for now.

Reply:
Yes, that would be great!

Reply:
PR created: https://github.com/zaproxy/zaproxy-website/pull/2100.